### DISCUSSION TITLE
ARIA UI Support - Audio Experience Only

### DISCUSSION THREAD

**Discussion by hollowaykeanho** [ZORALab/Hestia#259] [Q&A]
@corygalyna , could you work on the ARIA UI support specificially complete blindness user (audio-only) please? 

I still can't visualize how a blind individual get to a website and even use it without external assistances. For what I know, current ARIA support assumes audio-only experience.

I do have a hunch they're using a specialized hardware that helps them navigate the Internet. More research is required.

**Comment by corygalyna** [comment]
Acknowledged.

**Comment by corygalyna** [comment]
This is a huge project requiring additional testing as well. Like... we need to find a focus group of blind people operating on web.

**Comment by hollowaykeanho** [comment]
Need help?

**Comment by corygalyna** [comment]
Closing this task for the time being. Will reopen again once formalized.
